pkg/controller/beat/common/stackmon/metricbeat.tpl.yml

```diff
@@ -0,0 +1,9 @@
+metricbeat.modules:
+- module: beat
+  metricsets:
+    - stats
+    - state
+  period: 10s
+  hosts: ["{{ .URL }}"]
+
```

Comment:
Don't we need `xpack.enabeld: true` here according to https://www.elastic.co/guide/en/beats/metricbeat/current/monitoring-metricbeat-collection.html

Reply:
Upon testing, yes it was needed.  Added.